spring-security-modules/spring-security-oauth2-testing/reactive-resource-server/pom.xml

```diff
@@ -54,4 +56,9 @@
             </plugin>
         </plugins>
     </build>
```

Comment:
Why did we add this?

Reply:
Hi @anastasiosioannidis 

This was not added earlier it was already present I added the property in the end as per the format. 

  <groupId>com.baeldung.spring-security-modules.testing</groupId>
    <artifactId>reactive-resource-server</artifactId>
    <name>reactive-resource-server</name>
    <description>Demo project for Spring Boot reactive resource-server</description>
    <properties>
        <java.version>17</java.version>
    </properties>
    
I guess this is not needed anymore since this was removed recently in commit cb06a81ebc8ce5943cdd27c86586b67355da9d7e

Reply:
Could we remove ?

    <properties>
        <java.version>17</java.version>
    </properties>

Reply:
It is removed